### DISCUSSION TITLE
CVPR - Prework

### DISCUSSION THREAD

**Discussion by paularamo** [openvinotoolkit/openvino_notebooks#568] [Show and tell]
We will start this conversation with a big WELCOME to the CVPR community, we are pleased you have chosen this tutorial. Here, you will learn how to set up and run many AI models with OpenVINO and your webcam in just 10 minutes. As part of the tutorial, you should set up your laptop, in order to run the demonstration we have prepared for you. 

- Follow the instructions of this [Wiki page](https://github.com/openvinotoolkit/openvino_notebooks/wiki).
- Run the notebooks 301 and 401.
- Share a screenshot in this thread.

**Comment by pfvideo** [comment]
Screenshot of notebooks 301:


![image](https://user-images.githubusercontent.com/40673082/173221490-dc64ff3d-9749-46b8-b848-b3085fdcc365.png)

Notebooks 401:
![image](https://user-images.githubusercontent.com/40673082/173221550-a96fe90a-ff6a-4c38-89a5-3ed3f8d3ec94.png)

**Reply by paularamo** [reply]
Thanks @pfvideo. Could you please share the results after running the notebooks? 👍

**Comment by manduk-ai** [comment]
Hi, 
A pre-work was a great idea - thanks. 
Some of my screenshots.

301 training notebook - inference:
![301-training](https://user-images.githubusercontent.com/54119945/174067875-81fc8e2e-1c85-4548-9859-e6363459b53d.jpg)

******************

301 pot notebook: Accuracy looks the same for the compressed model:
![301-pot-1](https://user-images.githubusercontent.com/54119945/174067868-1428193c-7b11-4958-9c18-745b6634ec79.jpg)

******************

301 pot notebook - inference:
![301-pot-2](https://user-images.githubusercontent.com/54119945/174067873-b1c89ba8-05de-4936-a646-17f961dc9eca.jpg)

******************

401 notebook - a "knife" ;-), the algo seems a little bit over aggressive
![401-results-1](https://user-images.githubusercontent.com/54119945/174067876-75565bba-7a8b-4526-86ef-01872e4f614f.jpg)

******************

401 notebook - real time detection (camera). Sorry, I'm still a little bit shaken after yesterday's FOMC:
![401-results-2](https://user-images.githubusercontent.com/54119945/174067877-27f4a9b2-7169-454e-9cef-5b3bcb23a5e5.jpg)

******************

401 notebook - also ok with a video as a source
![401-results-3](https://user-images.githubusercontent.com/54119945/174067880-7de726ed-04e6-4a9a-b0fb-7177bbe6ee8f.jpg)

**Reply by manduk-ai** [reply]
Also: don't have a supported GPU so no data from the very interesting "Compare Inference Speed" part.

**Reply by paularamo** [reply]
Thanks @AIGeekProgrammer. Well done! See you on Sunday 👍

**Comment by mengjian0502** [comment]
Hi there, 

Nice pre-work! Does the OpenVino supports Pytorch models? 

Here're some of my screenshots: 
---
Notebook 301
---
![Screenshot from 2022-06-17 17-47-58](https://user-images.githubusercontent.com/35363367/174415734-1c4ea8a0-0550-40c3-ba0b-a503e760311a.png)

---
Notebook 301 - POT
---
![Screenshot from 2022-06-17 17-46-08](https://user-images.githubusercontent.com/35363367/174415714-b590bb4b-4ae0-46e6-ac96-04a9160268c3.png)

---
Notebook 401 - Video-based detection
---
![Screenshot from 2022-06-17 17-46-57](https://user-images.githubusercontent.com/35363367/174415722-18109628-7ffc-418a-aa39-183e6e7be9c1.png)

See you on Sunday!

**Reply by paularamo** [reply]
Hey, great job, thanks for sharing

**Comment by kilianyp** [comment]
Worked without any problems 👍 See you tomorrow
# 301 

<img width="1080" alt="image" src="https://user-images.githubusercontent.com/5173119/174461376-ce4c0dcc-c727-4096-8d2b-bab7d4541e84.png">


<img width="1080" alt="image" src="https://user-images.githubusercontent.com/5173119/174461369-84a449c0-94b9-4fdd-9ab1-fe3d315bc0bc.png">

# 401
<img width="842" alt="image" src="https://user-images.githubusercontent.com/5173119/174461159-b9533592-340d-45ff-a2c7-9adcaa7af6d5.png">

**Reply by paularamo** [reply]
See you soon, thanks for sharing!

**Comment by malamleh93** [comment]
301
<img width="1680" alt="image" src="https://user-images.githubusercontent.com/13084479/174494877-3527700b-03a7-4f62-803c-6ef87a47374a.png">



401

<img width="1249" alt="image" src="https://user-images.githubusercontent.com/13084479/174494828-86cd4261-fd40-4fec-92cf-59a760ad7d4d.png">

**Comment by hcarrillo** [comment]
Hi my  401   notebook result
<img width="1134" alt="Screen Shot 2022-06-19 at 1 24 33 PM" src="https://user-images.githubusercontent.com/2282970/174495150-ef7b686c-ea6d-402f-a7d6-25f83d9db4b1.png">

**Reply by hcarrillo** [reply]
Results of 301 notebook
<img width="1135" alt="Screen Shot 2022-06-19 at 1 27 17 PM" src="https://user-images.githubusercontent.com/2282970/174495242-5c5e1291-e0c8-47bc-a28f-b05bdb234c8d.png">

**Reply by raymondlo84** [reply]
Neat!

**Comment by TanguyJeanneau** [comment]
401
<img width="1289" alt="Screenshot 2022-06-19 at 14 02 12" src="https://user-images.githubusercontent.com/35469824/174496546-6ec54947-6cfe-41b9-b79e-f6fd6bf45013.png">
<img width="670" alt="image" src="https://user-images.githubusercontent.com/35469824/174496587-e5ac6bdf-7f83-4f83-aef7-ce385b328055.png">
---
301
Not all dataset seems to load for 301 though
<img width="664" alt="image" src="https://user-images.githubusercontent.com/35469824/174496724-05be20c4-570c-4d51-bf9b-b6b1c1d466b5.png">

**Reply by raymondlo84** [reply]
Oh! I remember you now! :) Thanks for coming again!

**Comment by rvainshtein** [comment]
Quick & dirty way to insert inference on a picture of a cat 😉
![image](https://user-images.githubusercontent.com/39670084/174496792-7d188c43-e46a-4aaf-bfbc-810e6f1cd54c.png)

**Comment by paularamo** [comment]
Thanks all for your participation in our tutorial!
